src/wp-includes/comment-template.php

```diff
@@ -2380,8 +2379,6 @@ function wp_list_comments( $args = array(), $comments = null ) {
 		$parsed_args['reverse_top_level'] = ( 'desc' === get_option( 'comment_order' ) );
 	}
 
-	wp_queue_comments_for_comment_meta_lazyload( $_comments );
```

Comment:
This function is now not used in core. Wonder if we should a note in the phpdocs for this function.

Reply:
If it's not used, maybe worth deprecating? I doubt it is seeing much use in plugins, so deprecating would seem reasonable to me.

Reply:
@peterwilsoncc Thoughts on deprecating This function?

Reply:
This docblock would need to be updated as part of the process

https://github.com/WordPress/wordpress-develop/blob/4cb3e1582a022e2d59ba3c4bf8dcb5c60c45eefc/src/wp-includes/class-wp-query.php#L4883-L4896

I did a search in the [plugin repo](https://wpdirectory.net/search/01GZZD0RE5AK0CMSVFVNSM6FZ3), the [theme repo](https://wpdirectory.net/search/01GZZD3C7CZYMXWEVMF0B840BP) and [on GitHub](https://github.com/search?q=wp_queue_comments_for_comment_meta_lazyload&type=code). It doesn't appear to be used a great deal so I think it's safe to deprecate.

There's a similarly named function `wp_queue_posts_for_term_meta_lazyload()` so there's some potential for confusion as to why one remains but not the other.

Reply:
@peterwilsoncc Should we deprecate this in this commit or in another ticket?

`wp_queue_posts_for_term_meta_lazyload` is a different thing, that is primes term meta from a array of posts.

Reply:
> Should we deprecate this in this commit or in another ticket?

I'm happy either way.